framework/include/fvkernels/FVPointValueConstraint.h

```diff
@@ -32,8 +32,10 @@ class FVPointValueConstraint : public FVScalarLagrangeMultiplierConstraint
   static InputParameters validParams();
 
   FVPointValueConstraint(const InputParameters & parameters);
+  virtual void meshChanged() override { setMyElem(); }
```

Comment:
pressurePin will probably want the same. Does it work better by any chance?

Reply:
You look for a new element every time you `execute` so `NSFVPressurePin` seems fine as is.

We don't really have any tests of this class with adaptivity, so my only reference point is the new laser welding input ... and yes with this change adaptivity definitely works better ... the singular row is no longer singular